src/main/java/uk/gov/hmcts/reform/sscs/helpers/FlagCodes.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.sscs.helpers;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+@SuppressWarnings("PMD")
+public enum FlagCodes {
+
+    RA0019("Step free / wheelchair access", 6),
```

Comment:
I would flip this around - instead having the enum values be the mappingValues in your helper. You can then have the code as a field on the enum. You shouldn't need anything else for this.

This makes it a lot more readable for us, as it stands RA0019 means nothing to us - a developer would have to come check the enum to see what it stands for

Reply:
This should be in commob

Reply:
I dont see why this should be in common as we are using common as a dependency and this will only expose endpoints. If I am missing something please enlighten me.

Reply:
> I would flip this around - instead having the enum values be the mappingValues in your helper. You can then have the code as a field on the enum. You shouldn't need anything else for this.
> 
> This makes it a lot more readable for us, as it stands RA0019 means nothing to us - a developer would have to come check the enum to see what it stands for

Implemented.... I this this is a good idea